library/gen_x12_837.inc.php

```diff
@@ -440,11 +367,12 @@ function gen_x12_837($pid, $encounter, &$log, $encounter_claim=false) {
   //   $log .= "*** CMS ID is missing for payer '" . $claim->payerName() . "'.\n";
   // }
 
-  if (true) { // !$CMS_5010
+  if (true) { 
```

Comment:
It is a block placeholder for a piece we really should revisit.  Maybe I should just wrap it in a todo set of comment lines.